src/App.jsx

```diff
@@ -1,21 +1,41 @@
 import Home from "./pages/Home";
 import TopicSelect from "./pages/TopicSelect";
 import { Routes, Route } from "react-router-dom";
-import Article from "./pages/Article";
+import Articles from "./pages/Articles";
 import Header from "./components/Header";
 import Nav from "./components/Nav";
 import User from "./components/user/User";
+import { useState } from "react";
+import ArticleFull from "./pages/ArticleFull";
 
 function App() {
+	const [topic, setTopic] = useState({});
+	const [topics, setTopics] = useState([]);
+	const [article, setArticle] = useState({});
+
 	return (
 		<div className="App">
 			<Header />
 			<Nav />
 			<User />
 			<Routes>
-				<Route path="/" element={<Home />} />
-				<Route path="/topic" element={<TopicSelect />} />
-				<Route path="/articles" element={<Article />} />
+				<Route path="/" element={<Home setArticle={setArticle} />} />
+				<Route
+					path="/topics"
+					element={
+						<TopicSelect
+							topic={topic}
```

Comment:
You shouldn't be sending props down into individual components within your router. Can you think of a hook we used to get the PARAMeter out of our endpoint? Then you can hold the article in state in the individual component.